velox/expression/tests/ExpressionFuzzerTest.cpp

```diff
@@ -38,7 +38,7 @@ DEFINE_string(
     special_forms,
     "and,or",
     "Comma-separated list of special forms to use in generated expression. "
-    "Supported special forms: and, or, coalesce, if.");
+    "Supported special forms: and, or, coalesce, if, switch, cast.");
```

Comment:
Are you able to clear all the test failures after enabling cast? I'm trying it locally too but still run into some errors.

Reply:
Never mind, I misunderstood it.